include/dlr_common.h

```diff
@@ -113,6 +113,8 @@ std::string GetStringFromDeviceType(DLDeviceType device_type);
 
 DLDeviceType GetDeviceTypeFromMetadata(const std::vector<std::string>& model_paths);
 
+std::vector<std::string> MakePathVec(std::string model_path);
```

Comment:
`std::string model_path` -> `const std::string& model_path`

Reply:
this cannot be reference since we are passing char * as input.